images/macos/tests/Common.Tests.ps1

```diff
@@ -48,33 +48,134 @@ Describe "Audio device" -Skip:($os.IsHighSierra -or $os.IsBigSur) {
     }
 }
 
-Describe "Common utilities" {
-    It "Homebrew" {
-        "brew --version" | Should -ReturnZeroExitCode
+Describe "CommonUtils" {

```

Comment:
On Windows and Ubuntu, we follow approach that tool name is presented as Describe and It is used for test cases. Do we want to use consistent approach of MacOS?
https://github.com/actions/virtual-environments/blob/main/images/win/scripts/Tests/CLI.Tools.Tests.ps1

Reply:
Will be fixed in the next PRs